app/assets/stylesheets/elements/_table.css

```
@@ -0,0 +1,24 @@
+.user-table {
```

Seguindo a convenção BEM, esse nome de classe é errado, não é?
```suggestion
.user--table {
```

---

mas no caso ele não é um modifier, ele é um nome composto mesmo, a minha table é o meu Block, que dei esse nome composto de "user-table", e os elementos dele eu usei como "user-table__elements", eu usei assim pq procurei e vi que  quando se usa nome composto se separa por hífen, mas não sabia se poderia usar no BEM